### PR TITLE
Build protoc in Travis

### PR DESCRIPTION
**Pull request by ejona86**
This mostly fixes the Travis-CI build

### CODE REVIEW

**Comment by ejona86** [comment]
@zhangkun83, would you mind reviewing this? I know it's in a strange repository :)

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,23 @@
+sudo: false
+
 language: java
+
+notifications:
+  email: false
+
+env:
+  global:
+    - PATH="/tmp/proto2.4.1/bin:$PATH"
```
Comment (zhangkun83) [inline .travis.yml]: Why not use $PROTO_VERSION here and in the below lines where 2.4.1 is used?

Comment (ejona86) [inline .travis.yml]: Okay. I've added PROTO_VERSION as an environment variable in the .travis.yml and use it instead of duplicating the version.

**Comment by zhangkun83** [comment]
LGTM

**Comment by ejona86** [comment]
Merged as 3cc6568

**Pull request closed without merging**